Reference topic (Task category) (#82)
UWC2-APIDOC/to-do-service-public: docs/api/tasks-category.md

---
layout: page
---

# Tasks category

> Returns an array of  [`task`](task) objects that contains tasks that have the specified `category` parameter, if it exists.

Reviewer: ```suggestion
Returns an array of  [`task`](task) objects that contain the specified `category` parameter, if any exist.
```
Simplified some.
